### ISSUE TITLE
Wikipreview unable to produce working links to sections

### ISSUE THREAD

**Issue by ARLundborg** [Frederisk/Wikitext-VSCode-Extension#25]
Suppose I wanted to link to the section "Position in the week" on the "Monday" wikipedia page on the sentence "Monday is either the first or the last day of the week." I would use the syntax:
```
[[Monday#Position_in_the_week| Monday is either the first or the last day of the week.]]
```
which is working on the main Wiki site. However, if I do this in VScode viewer and use the preview, the link is not clickable. The links are clickable if you do not link to a section but merely to a page.

**Comment by Frederisk** [comment]
I investigated the issue and found that this seems to be a strange feature from VSCode. I am not sure if this is a Bug from VSCode, I need more time to understand how it works.🤔

**Comment by Frederisk** [comment]
Continue here: microsoft/vscode#121045

**Comment by Frederisk** [comment]
Fixed: microsoft/vscode/commit/be2026c324c3dae54680329e52e24cd95dff6718
